app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+from app.errors import (
+    OutdatedVaccineError,
+    NotVaccinatedError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> str:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> None:
+        if not ("vaccine" in visitor):
+            raise NotVaccinatedError()
+        elif datetime.date.today() > visitor["vaccine"]["expiration_date"]:
```

Comment:
Before accessing 'visitor["vaccine"]["expiration_date"]', ensure that 'expiration_date' exists in the 'vaccine' dictionary to avoid potential KeyError.